Description: Add shebang (fixes lintian warning)
 executable-not-elf-or-script [usr/share/cargo/registry/triple_accel-0.4.0/build_ir_asm.sh]
Origin: self
Forwarded: https://github.com/Daniel-Liu-c0deb0t/triple_accel/issues/13
--- a/build_ir_asm.sh
+++ b/build_ir_asm.sh
@@ -1 +1,2 @@
+#!/bin/sh
 RUSTFLAGS="--emit llvm-ir,asm" cargo build --release
